liquibase-core/src/main/java/liquibase/snapshot/jvm/SequenceSnapshotGenerator.java

```diff
@@ -242,6 +243,32 @@ protected String getSelectSequenceSql(Schema schema, Database database) {
                     "FROM SYS.SYSSEQUENCE s " +
                     "JOIN SYS.SYSUSER u ON s.OWNER = u.USER_ID "+
                     "WHERE u.USER_NAME = '" + schema.getName() + "'";
+        } else if (database instanceof MariaDBDatabase) {
+          StringJoiner j = new StringJoiner(" \n UNION\n");
+          try {
+            List<Map<String, ?>> res = ExecutorService.getInstance().getExecutor(database)
+            .queryForList(new RawSqlStatement("select SUBSTRING( NAME FROM LOCATE( '/' ,NAME ) + 1 ) AS SEQUENCE_NAME "
+                + "from information_schema.INNODB_SYS_TABLES "
+                + "where NAME like '" + schema.getName() +"/%' AND FLAG & 12288 ;"));
+            // see https://mariadb.com/kb/en/information-schema-innodb_sys_tables-table/#flag for 12288 value of (3 << 12)
+            if (res.size() == 0) {
+              return "SELECT name AS SEQUENCE_NAME WHERE 1=0";
```

Comment:
I think this line shoulb be `return "SELECT '" + schema.getName() +"' AS SEQUENCE_NAME WHERE 1=0";`

Reply:
I thought that, too, but was reluctant to make the change.  It seems like it was that same literal in other sequence code.  I'll check again.